tsconfig.json

```diff
@@ -19,7 +19,7 @@
 
     /* JavaScript Support */
     "allowJs": true /* Allow JavaScript files to be a part of your program. Use the 'checkJS' option to get errors from these files. */,
-    "checkJs": false /* Enable error reporting in type-checked JavaScript files. */,
+    "checkJs": true /* Enable error reporting in type-checked JavaScript files. */,
```

Comment:
Should this be false until we can rewrite the JS? Or can we flag the problems but show them as warnings instead of errors? I don't like all the errors in my IDE and I felt like it stopped a multi-step build from progressing past the transpilation step.

Reply:
Makes sense to me. I think most of the errors just amount to complains about lack of non-null and non-undefined checks and implicitly any types, all of which can be dealt with during the conversation process.